python/lsst/analysis/tools/interfaces/datastore/_sasquatchDatastore.py

```diff
@@ -110,6 +120,16 @@ def __init__(
         else:
             self.namespace = self.config.get("namespace", "lsst.dm")
 
+        extra_fields: dict[str, str | int | float] | None = None
+        if extras_str := os.environ.get("SASQUATCH_EXTRAS"):
+            extra_fields = {}
+            for item in extras_str.split(";"):
+                k, v = item.split("=")
+                extra_fields[k] = v
+            self.extra_fields = extra_fields
+        else:
+            self.extra_fields = self.config.get("extra_fields", extra_fields)
```

Comment:
On this branch, `extra_fields` is always `None`... was that intentional?

Reply:
Yes, I intended `None` there because that was the originally default for no fields and is what the dispatcher wants (it explicitly checks for None). I had not considered merging but that seems like a reasonable thing to do since these are not hierarchical.